contribution/mobile/ja/procedural-names.json

```diff
@@ -250,6 +245,7 @@
 				"ライフル",
 				"キャノン砲",
 				"ザッパー"
+				"火炎放射器",
```

Comment:
Add a comma (on line 247), and erase the comma (on line 248) as it's the last variable in the array.

Reply:
Yes. Your changes were okay now.
You can see if the changes were correct or not by looking at the Check Pass below (green = good to go/pass the check).

As for the next question, you just need a bare minimum knowledge of code language (particularly JSON & Markdown) in order to do a PR change. Of course, it's normal to see a mistake / mistype whenever someone did a PR, as verifier/repo owner will tell you which part was the issue & will encourage you to fixes the issue (or in other cases, the verifier/repo owner will do these fixes & push it to the branch immediately). So.. nope. it didn't bother me (if that question were aim towards my previous comment) or other verifier/repo owner :)

Reply:
I'm relieved to hear that! This is my first time to join such community so I was worrying. Thank you all! :)